### ISSUE TITLE
how to train?

### ISSUE THREAD

**Issue by kimtaehyeong** [guoxih/region-based-non-local-network#1]
I am very interested in your research.

I want to perform custom learning on my data, but I am curious about the example training code and how to organize the data.

If this is not an excuse, I would like you to provide the code to create the dataset.

Thank you.

**Comment by edhuang1** [comment]
For the training on Something-V1 dataset.  I use the script below:
python main.py --dist-url 'tcp://localhost:6666' --dist-backend 'nccl' --multiprocessing-distributed --available_gpus 0,1,2,3 --world-size 1 --rank 0 --dataset something --gd 20 --shift --shift_div=8 --shift_place=blockres --npb --lr 0.02 --wd 1e-3 --dropout 0.8 --num_segments 8 --batch_size 16 --batch_multiplier 4 --use_warmup --warmup_epochs 1 --lr_type cos --epochs 50 --non_local  --suffix 1
you should get the same result on the paper.  The trained models will release recently.

About how to organize data, the code of my data is acturally based on [https://github.com/yjxiong/tsn-pytorch](url), which can provide you more details.

**Comment by kimtaehyeong** [comment]
Thank you so much!
